data/mods/aftershock_exoplanet/Map/overmap_terrain/ice_fields.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "overmap_terrain",
+    "id": "field",
+    "copy-from": "generic_open_land",
+    "name": "Ice Fields",
+    "sym": ".",
+    "color": "light_gray",
+    "see_cost": 2,
+    "extras": "field",
+    "mapgen": [ { "method": "builtin", "name": "field" } ],
+    "flags": [ "NO_ROTATE" ]
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "forest",
+    "copy-from": "generic_forest",
+    "name": "hydrothermal flats",
+    "sym": "%",
+    "color": "light_red",
+    "see_cost": 3,
+    "extras": "forest",
+    "spawns": { "group": "GROUP_FOREST", "population": [ 0, 3 ], "chance": 13 },
+    "flags": [ "NO_ROTATE", "SOURCE_FORAGE" ]
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "forest_thick",
+    "copy-from": "generic_forest",
+    "name": "hydrothermal flats",
+    "sym": "%",
+    "color": "light_red",
+    "see_cost": 4,
+    "extras": "forest_thick",
+    "spawns": { "group": "GROUP_FOREST", "population": [ 0, 6 ], "chance": 15 },
+    "mapgen": [ { "method": "builtin", "name": "forest" } ],
+    "flags": [ "NO_ROTATE", "SOURCE_FORAGE" ]
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "forest_water",
+    "copy-from": "generic_wetland_forest",
+    "name": "hydrothermal flats",
```

Comment:
Same name? What do you have in mind for this terrain?

Reply:
This one would be warmer, wetter and more livier than the other. Not super sure what to call it.

Reply:
I'm not great with names - maybe simply "warm hydrothermal flats"?